src/decoder/mod.rs

```diff
@@ -780,7 +780,6 @@ impl fmt::Display for DecoderError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let text = match self {
             DecoderError::UnrecognizedFormat => "Unrecognized format",
-            #[cfg(feature = "symphonia")]
```

Comment:
It's not just Symphonia that uses this variant. I started trying to fix up the decoder to work with no formats, but there's no point, so switched to feature gating.